fdbclient/tests/fdb_cluster_fixture.sh

```diff
@@ -58,7 +58,7 @@ function start_fdb_cluster {
       "${local_build_dir}" \
       --knobs "${knobs}" \
       --stateless_count 1 --replication_count 1 --logs_count 1 \
-      --storage_count "${ss_count}" --storage_type ssd \
+      --storage_count "${ss_count}" --storage_type ssd-rocksdb-v1 \
```

Comment:
Do we want to randomly choose between sqlite and rocksdb? for coverage

Reply:
Dunno. rocksdb is the standard, not sqllite.

Reply:
We want bulkload feature to stably work for both sqlite and rocksdb